widget/embedded/src/pages/WalletsPage.tsx

```diff
@@ -146,13 +149,21 @@ export function WalletsPage({
             sortedWallets.map((wallet, index) => {
               const key = `wallet-${index}-${wallet.type}`;
               return (
-                <Wallet
-                  {...wallet}
-                  key={key}
-                  onClick={(type) => {
-                    void onSelectWallet(type);
-                  }}
-                />
+                <>
+                  <Wallet
+                    {...wallet}
+                    key={key}
+                    onClick={(type) => {
+                      void onSelectWallet(type);
+                    }}
+                  />
+                  <WalletModal
+                    open={openModal === wallet.type}
```

Comment:
I think it's better when the wallet is connected, it shows the connection modal for a few seconds and then close.

Reply:
fixed.